lib/active_zuora/generator.rb

```diff
@@ -196,8 +196,7 @@ def add_extra_customizations
       customize 'RatePlanCharge' do
         include LazyAttr
         exclude_from_queries :overage_price, :included_units,
-          :discount_amount, :discount_percentage, :rollover_balance, :price
```

Comment:
Agree we should avoid forking the gem. I already raised the [AR5 compatibility issue](https://github.com/sportngin/active_zuora/issues/74) on the main repo for awareness. If/when it bumps compatibility version, I think that's the time we go back and re-record this cassette and dump the fork.

Will add comments per directives here. Thanks for the review.